README.md

```diff
@@ -711,9 +711,9 @@ needed. For more details see "Skipping empty values" section.
 
 ```php
 use Yiisoft\Validator\Rule\Required;
-use Yiisoft\Validator\SkipOnEmptyCallback\SkipOnNull;
+use Yiisoft\Validator\EmptyHandler\NullEmpty;
 
-$rules = [new Required(emptyCallback: new SkipOnNull())];
+$rules = [new Required(emptyCallback: new NullEmpty())];
```

Comment:
There are more places left with old names, this for example:

https://github.com/yiisoft/validator/blob/8b20d2b2c99c2129aa66f3606e519e728c5de339/README.md?plain=1#L160